tsl/src/remote/connection.c

```diff
@@ -289,7 +289,15 @@ fill_result_error(TSConnectionError *err, int errcode, const char *errmsg, const
 	const char *sqlstate;
 
 	if (NULL == err || NULL == res || NULL == entry)
+	{
+		if (NULL != err)
+		{
+			/* err should be filled unless we return true */
```

Comment:
Does this function ever return `true`?

Reply:
Good point, thanks. I changed the comment.

Reply:
On that point, if it always returns false, maybe it should become `void`? Maybe it needs some refactoring.